instaclustr/utils.go

```diff
@@ -0,0 +1,16 @@
+package instaclustr
+
+import (
+	"encoding/json"
+)
+
+func StructToMap(obj interface{}) (newMap map[string]interface{}, err error) {
+	data, err := json.Marshal(obj)
+
+	if err != nil {
+		return
+	}
+
+	err = json.Unmarshal(data, &newMap)
+	return
+}
```

Comment:
**[Note]** This really isn't an efficient solution, unmarshalling stuff on the fly isn't ideal, a sturdier implementation would still access the tagging attributes via reflection on a recursive deconstruction/mapping for an arbitrary struct. Though there is not necessity here as the overhead is minimal. Just a though to keep in mind